src/main.ts

```diff
@@ -78,8 +78,8 @@ async function start () {
   await KeycloakService.setKeycloakConfigUrl(keycloakConfig)
 
   // initialize token service which will do a check-sso to initiate session
-  // don't start during Jest tests as it messes up the test JWT
-  if (process.env.JEST_WORKER_ID === undefined) {
+  // don't start during Vitest tests as it messes up the test JWT
+  if (import.meta.env.VITEST_WORKER_ID === undefined) {
```

Comment:
When I tested this change locally, I just used
```
if (!import.meta.env.VITEST) {
```

Reply:
From https://vitest.dev/guide/migration.html:
![vitest envs](https://github.com/bcgov/business-create-ui/assets/122301442/d2b30f6b-5b53-431f-9972-0ee68d35dc80)

Reply:
Well.... maybe we shouldn't have been looking at the Jest ID before, and maybe it's better to look for VITEST='true' now...?

Reply:
So I've tested with `VITEST` instead of using the worker ID, all tests pass. I agree with you. I'll be changing that.